client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -78,9 +80,16 @@ function WPLineItem( {
 		sublabelAndIntervalPriceBreakdown = item.sublabel;
 	}
 
+	const productSlug = item.wpcom_meta?.product_slug;
+	const isBusinessPlanProduct = productSlug && isBusinessPlan( productSlug );
+	const productName =
+		isBusinessPlanProduct && isWhiteGloveOffer
+			? `${ item.label } (White glove edition)`
```

Comment:
Does "White glove edition" need to be translated for other locales?

Reply:
We are running this test EN-US to begin with, so we will revisit translations if we later find the need to launch to other locales.